tokenList.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "chainId": 3222583904,
```

Comment:
this is not applicable to other chains... i don't think we should include this file in this repo (we can always show an example in the README)